### DISCUSSION TITLE
[Question] Changing landscape generation

### DISCUSSION THREAD

**Discussion by Nyasha1234** [Roadhog360/Et-Futurum-Requiem#456] [Ideas]
Minecraft 1.17 surprised players with a new generation of caves, which are very beautiful, but not very convenient for speleology. Minecraft 1.18 has significantly changed terrain generation for the first time since version 1.7.10, making it more mountainous. If 1.17 was a controversial version, in my opinion (there was no need to increase the height of the world for the sake of huge caves, since they could be made a little smaller within 256 blocks), then 1.18 was very successful. Beautiful mountains, a new river generation system, and hills made Minecraft more realistic and pleasing to look at.

In one of the threads I noticed that you do not want to add new biomes so as not to change the 1.7.10 seed. I'm far from mods and programming, but I would like to know - why is this necessary? Wouldn't it be more pragmatic to try to change the 1.7.10 seed as much as possible so that it is similar to the 1.18-1.20 seed? I really don't know if this is even possible.

If it is possible to change the 1.7.10 landscape to a more modern one (1.18), then it would look just great. Because the argument of many players of the new versions is precisely that modern Minecraft looks more beautiful.

Sorry for my naive questions, I don't understand Minecraft terrain generation methods. Thank you for supporting my favorite version!

**Comment by Roadhog360** [comment]
Unfortunately I don't really understand how terrain gen or noise works.

**Comment by Roadhog360** [comment]
At least someone agrees that changing the height isn't needed. Possible but not feasible and I wouldn't be doing it. I do plan on some cave content at least.

For seeds I simply do not want to introduce harsh chunk borders. But adding a new biome would not really produce seeds closer to 1.17 anyways. Updates before 1.17 were able to introduce biomes without disturbing seeds just fine to despite that people will tell you it's not possible, it is, it's just no one's bothered to figure out how since breaking seed parity is much easier.

**Comment by Nyasha1234** [comment]
> Possible but not feasible and I wouldn't be doing it

Sadly, this is a very beautiful landscape. 

> I do plan on some cave content at least.

Are you planning to add noise caves, dripstone and lush caves, aquifers? There is a now-abandoned addon for GT6 - Greggen, which partially implemented this. But it's worth noting that this addon needs to be improved.

**Reply by Roadhog360** [reply]
Again, I'm not very familiar with World generation or noise generation, so the backport would just be in a modified version of the existing caves.
